web/public/js/init-master.js

```diff
@@ -1045,12 +1046,35 @@
         }
         pageTitle.push("CoderDojo Zen");
         $rootScope.pageTitle = pageTitle.join(" | ");
+
+        //if moving from another page
+        if (!fromState.abstract) {
```

Comment:
abstract means it has a superState to which you can't navigate : it should always be false, basically.
Instead, maybe try $cookie.get('cookieDisclaimer') to verify it's existing or not
Elsewhat i'll be forever refreshed, meaning the cookie TTL will be.. well, non-respected.